### PR TITLE
[HIRFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by jomlink**

### CODE REVIEW

File: submissions/jonasMwansa/database/transportation_db.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+id INT NOT NULL AUTO_INCREMENT,
+full_name VARCHAR(255) NOT NULL,
+sex VARCHAR(10) NOT NULL,
+age INT NULL,
+PRIMARY KEY(id)
+);
+
+/*create a passengers class table with the associated properties*/
+CREATE TABLE passengers_class_table(
+id INT NOT NULL AUTO_INCREMENT,
```
Comment (ToluEnny24) [inline submissions/jonasMwansa/database/transportation_db.sql:16]: the passengers class ticket and other details should be on the same table.

File: submissions/jonasMwansa/database/transportation_db.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+passenger_id INT NOT NULL,
+ticket_id INT NOT NULL, 
+fare DECIMAL(20,4) NOT NULL,
+cabin VARCHAR(255) NULL,
+number_of_parents_or_children INT NULL, 
+number_of_siblings_or_spouses INT NULL,
+point_of_embarkation VARCHAR(255)  NOT NULL,
+
+PRIMARY KEY(id),
+FOREIGN KEY(passenger_class_id) REFERENCES passengers_class_table(id),
```
Comment (ToluEnny24) [inline submissions/jonasMwansa/database/transportation_db.sql:46]: This would be removed when you fix the table

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
@jomlink Welldone effect all the requested changes

**Review by ToluEnny24 — APPROVED** [review]
Well done @jomlink

**Comment by jomlink** [comment]
Thank you madam

**Pull request merged**